src/sentry/search/snuba/backend.py

```diff
@@ -89,6 +88,10 @@ def _query(self, projects, retention_window_start, group_queryset, environments,
                         groupenvironment__first_release__version=version,
                     )
                 ),
+                'first_seen': ds.ScalarCondition(
```

Comment:
Just curious what the SQL ends up looking like with this? I assume we end up joining to the table twice, but there might not be a great way to avoid that.